### PR TITLE
Implementation of clearAmbientCache functionality

### PR DESCRIPTION
**Pull request by danieljosua1**

### CODE REVIEW

**Comment by danieljosua1** [comment]
@m0nac0

File: maplibre_gl_platform_interface/lib/src/maplibre_gl_platform_interface.dart [hunk 1]
```diff
@@ -74,6 +74,8 @@ abstract class MapLibrePlatform {
   Future<List> querySourceFeatures(
       String sourceId, String? sourceLayerId, List<Object>? filter);
   Future invalidateAmbientCache();
+
```
Comment (Zverik) [inline maplibre_gl_platform_interface/lib/src/maplibre_gl_platform_interface.dart]: I think this empty line is not needed (or should be moved below).

File: maplibre_gl/ios/maplibre_gl/Sources/maplibre_gl/MapLibreMapController.swift [hunk 2]
```diff
@@ -158,6 +158,15 @@ class MapLibreMapController: NSObject, FlutterPlatformView, MLNMapViewDelegate,
                     result(nil)
                 }
             }
+        case "map#clearAmbientCache":
+                    MLNOfflineStorage.shared.clearAmbientCache {
```
Comment (Zverik) [inline maplibre_gl/ios/maplibre_gl/Sources/maplibre_gl/MapLibreMapController.swift]: Please fix the indentation.

**Review by Zverik — COMMENTED** [review]
Looks fine to me, apart from some whitespace. Just duplicates the `invalidateAmbientCache` calls with a new name.

**Comment by danieljosua1** [comment]
@josxha @Zverik Thanks a lot for your review! I have adjusted the code accordingly 👍

**Review by Zverik — APPROVED** [review]

**Review by josxha — APPROVED** [review]
Thanks for the pull request @danieljosua1 and thanks for the review @Zverik!

The changes look good for me, too.

**Pull request merged**